test/mysql-chassis.js

```diff
@@ -316,6 +326,10 @@ describe('mysql-chassis', () => {
 
   // })
 
+  // describe('transaction methods', () => {
+
```

Comment:
Hi again,
I added the tests. They basically check if the chassis' methods call mysqljs' methods.
The way it works on the mysqljs' side is that all 3 methods (`beginTransaction`, `commit`, `rollback`) simply call `query` with `START TRANSACTION`, `COMMIT`, `ROLLBACK` arguments respectively. 
Is there anything else that should be tested?